src/PageContainer.js

```diff
@@ -19,6 +20,13 @@ class PageContainer extends Component {
 
         this.state = {
             content: props.content,
+            showPopUp: false,
+            hover: false,
+            // TODO: how to handle multiple notifications
+            notification: {
+                type: 'recurring_pickup_request',  // Enums.NotificationType
+                content: '-L5QoXeC_UrL5tRRED3e'  // key of DeliveryRequest
+            },
```

Comment:
- Each account will have a list of notifications in their account obj in the db, that looks like the below. So I'd change the state to be a list of notifications `notifications: []`.
```
notifications: [
    {
        type: 'recurring_pickup_request',
        content: '-L5QoXeC_UrL5tRRED3e'
    },
    {
        type: 'recurring_pickup_confirmed',
        content: '-XKSIDLeC_Uksd321e'
    },
    ...
]
```
- can you do `notification: []` here and move the mocking dummy data into `componentDidMount()` with a backend TODO comment?